diff --git a/node_modules/nextra/dist/loader.mjs b/node_modules/nextra/dist/loader.mjs
index 1e145f5..dbec997 100644
--- a/node_modules/nextra/dist/loader.mjs
+++ b/node_modules/nextra/dist/loader.mjs
@@ -611,6 +611,7 @@ function compileMdx(_0) {
   }, resourcePath) {
     var _a;
     let structurizedData = {};
+    let extraOptions = mdxOptions.rehypePrettyCodeOptions ? mdxOptions.rehypePrettyCodeOptions : {}
     const compiler = createCompiler({
       jsx: (_a = mdxOptions.jsx) != null ? _a : true,
       outputFormat: mdxOptions.outputFormat,
@@ -625,7 +626,7 @@ function compileMdx(_0) {
       rehypePlugins: [
         ...mdxOptions.rehypePlugins || [],
         parseMeta,
-        [rehypePrettyCode, rehypePrettyCodeOptions],
+        [rehypePrettyCode, {...rehypePrettyCodeOptions, ...extraOptions}],
         attachMeta
       ].filter(Boolean)
     });
